scripts/rhel_validation/prepare_lis_vm.sh

```diff
@@ -48,9 +65,18 @@ function install_modules {
     printf "\nKMOD Install: \n"
     printf "\nKMOD Install: \n" 1>&2
     rpm -ivh "$(ls kmod* | grep -vE 'i686')"
+    ret=$?
+    if [[ $ret -ne 0 ]];then
+	    printf "\n$(ls kmod* | grep -vE 'i686') installation failed \n"
```

Comment:
what is the reason in the i686 check?

Reply:
Just displaying what was failed to install. Above line identifies the RPM which needs to be installed.